lib/analyzer/applications/php.ts

```diff
@@ -0,0 +1,99 @@
+import * as lockFileParser from "@snyk/composer-lockfile-parser";
+import { legacy } from "@snyk/dep-graph";
+import * as path from "path";
+import { DepGraphFact, TestedFilesFact } from "../../facts";
+
+import { DepTreeDep } from "../../types";
+import { AppDepsScanResultWithoutTarget, FilePathToContent } from "./types";
+
+interface ManifestLockPathPair {
+  manifest: string;
+  lock: string;
+}
+const PACKAGE_MANAGER_TYPE = "composer";
```

Comment:
Is this how we usually add constants? Would it be an overkill to have it as an environment variable? Or maybe as part of some config/consts file?

Reply:
This is mostly to avoid putting this string directly when it's used in a function call. It's not expected to change, but I believe this way is a bit nicer and easier to read or extend in the future. It will be definitely be an overkill to add as an environment variable 🙂